aptos-node/src/main.rs

```diff
@@ -72,8 +72,17 @@ fn main() {
             rng,
         );
     } else {
-        let config = NodeConfig::load(args.config.unwrap()).expect("Failed to load node config");
+        // Load the config file
+        let config_path = args.config.unwrap();
+        let config = NodeConfig::load(config_path.clone()).unwrap_or_else(|error| {
+            panic!(
+                "Failed to load node config file! Given file path: {:?}. Error: {:?}",
+                config_path, error
```

Comment:
I guess this solves the immediate problem of "it's a problem with the config", but we should make a later story on how to tell what is wrong with the config, since serde doesn't help much with it

Reply:
Yeah, I think that'll require a more thorough story.